llama-cli/src/main.rs

```diff
@@ -0,0 +1,102 @@
+use std::io::Write;
+
+use cli_args::CLI_ARGS;
+use llama_rs::InferenceParams;
+use rand::thread_rng;
+
+mod cli_args;
+
+fn main() {
+    env_logger::init();
```

Comment:
We should make the `info` log level the default for the CLI app. The output doesn't look too noisy and model loading times are large enough that most users are going to want to see some progress indication, especially important for the first-time experience.
```suggestion
    env_logger::Builder::from_env(env_logger::Env::default().default_filter_or("info")).init();
```

Doing this allows setting the `RUST_LOG` environment variable to a higher level (like `warn` or `error`) so users who don't want to see the logs still get the choice to do so.